app/scripts/controllers/tables.js

```diff
@@ -106,6 +111,29 @@ angular.module('tables', ['stats', 'sql', 'common', 'tableinfo'])
       $scope.renderSiderbar = true;
       $scope.isParted = false;
 
+      var constructQuery = function (rows) {
+        var result = 'SELECT ';
+        var current_object;
+
+        for (var i = 0; i < rows.length; i++) {
+          if (current_object !== '' && rows[i].data_type === 'object' && 
+            !rows[i].column_name.startsWith(current_object)) {
+            current_object = rows[i].column_name;
+            result += '\"' + rows[i].column_name + '\", ';
+          }
+        
+          if (current_object && rows[i].column_name.startsWith(current_object)) {
```

Comment:
I think this logic does not work correctly. imagine a table with the following schema:

```sql
CREATE TABLE t1 (a OBJECT, ab OBJECT, abc OBJECT);
```